src/main/java/uk/gov/pay/products/resources/PaymentResource.java

```diff
@@ -46,6 +46,14 @@ public PaymentResource(PaymentFactory paymentFactory, PaymentRequestValidator re
         this.paymentFactory = paymentFactory;
         this.requestValidator = requestValidator;
     }
+    
+    @Path("/v1/api/payments/redact-reference/{govukPaymentId}")
```

Comment:
I made the govukPaymentId rather than externalId the search term here as this endpoint is meant to be called from toolbox and we definitely have the govukPaymentId (or what ledger calls resourceExternalId) there. Not sure if the externalId is readily available from the toolbox end.